Upstream status: Taken from upstream development.

From aace425e41247c1dd6b16eb7eabce50be7310d15 Mon Sep 17 00:00:00 2001
From: Janneke Nieuwenhuizen <janneke@gnu.org>
Date: Mon, 24 Apr 2023 16:58:32 +0200
Subject: [PATCH] DRAFT test: Add missing shebangs to run scripts.

This fixes `make check' on current ci.guix.gnu.org.

* test/all/parse_import_order/run,
test/all/parse_locations/run,
test/all/parse_peg_locations/run: Add #! /bin/sh.
---
 test/all/parse_import_order/run  | 3 ++-
 test/all/parse_locations/run     | 3 ++-
 test/all/parse_peg_locations/run | 3 ++-
 3 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/test/all/parse_import_order/run b/test/all/parse_import_order/run
index 6e231de4f9..c383e4f223 100755
--- a/test/all/parse_import_order/run
+++ b/test/all/parse_import_order/run
@@ -1,6 +1,7 @@
+#! /bin/sh
 # Dezyne --- Dezyne command line tools
 #
-# Copyright © 2022 Jan (janneke) Nieuwenhuizen <janneke@gnu.org>
+# Copyright © 2022, 2023 Jan (janneke) Nieuwenhuizen <janneke@gnu.org>
 #
 # This file is part of Dezyne.
 #
diff --git a/test/all/parse_locations/run b/test/all/parse_locations/run
index 0b092b26dd..6c48ee1392 100755
--- a/test/all/parse_locations/run
+++ b/test/all/parse_locations/run
@@ -1,7 +1,8 @@
+#! /bin/sh
 # Dezyne --- Dezyne command line tools
 #
 # Copyright © 2021 Paul Hoogendijk <paul@dezyne.org>
-# Copyright © 2020, 2021 Jan (janneke) Nieuwenhuizen <janneke@gnu.org>
+# Copyright © 2020, 2021, 2023 Jan (janneke) Nieuwenhuizen <janneke@gnu.org>
 #
 # This file is part of Dezyne.
 #
diff --git a/test/all/parse_peg_locations/run b/test/all/parse_peg_locations/run
index 5b5ced0bb4..5206a23db6 100755
--- a/test/all/parse_peg_locations/run
+++ b/test/all/parse_peg_locations/run
@@ -1,6 +1,7 @@
+#! /bin/sh
 # Dezyne --- Dezyne command line tools
 #
-# Copyright © 2020 Jan (janneke) Nieuwenhuizen <janneke@gnu.org>
+# Copyright © 2020, 2023 Jan (janneke) Nieuwenhuizen <janneke@gnu.org>
 #
 # This file is part of Dezyne.
 #
-- 
2.39.2

